lib/experimental/subject.rb

```diff
@@ -12,8 +12,8 @@ def experiment_bucket(name)
       Experimental.source[name].try { |e| e.in?(self) ? e.bucket(self) : nil }
     end
 
-    def in_bucket?(name, bucket)
-      in_experiment?(name) && experiment_bucket(name) == bucket
+    def in_bucket?(name, bucket, *args)
```

Comment:
What if we allowed the `bucket` param to be either an array or single value, and if it's an array we splat it internally?

or..could make an `in_buckets?` (plural) which takes an array, and they both call the same internal method?